fireant/database/redshift.py

```diff
@@ -16,3 +16,6 @@ def __init__(self, host='localhost', port=5439, database=None,
         super(RedshiftDatabase, self).__init__(host, port, database, user, password,
                                                max_processes=max_processes,
                                                cache_middleware=cache_middleware)
+
+    def get_column_definitions(self, schema=None, table=None):
```

Comment:
This should be removed if the aim is to fall back to the `PostgreSQLDatabase` `get_column_definitions` functionality

Reply:
Yes, you're right! I'll update it.